portal-impl/src/com/liferay/portal/change/tracking/sql/CTSQLContextFactory.java

```diff
@@ -17,7 +17,7 @@
 import java.util.List;
 
 /**
- * @author Preston Crary
+ * @author     Preston Crary
```

Comment:
this does not look good

Reply:
This changes were made with the formatJavadoc task in portal-impl. I doubted about only commit my changes or all the changes that the task made.

I can rebase and apply only the changes on the PR files to avoid creating noise with other non related files

Reply:
No, Ok. It's a strange formatting and a strange momento to apply it.